src/priority-queue-original.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * A priority queue stores a list of items but each can have a numeric priority value.
+ * Items with a higher priority are dequeued before items with a lower priority.
+ * Implemented as a hash of arrays where the hash keys are priority values.
+ */
+function PriorityQueue(size) {
```

Comment:
```suggestion
function PriorityQueue() {
```
Did you mean to leave this in? Should remove `size` if you aren't going to use it.